packages/new-bot-web-ui/src/components/blocks/blocks-components/blocks-fields.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { SelectBlockField } from '.';
+
+const BlocksFields = ({
+    markets_dropdown,
+    submarkets_dropdown,
+    symbols_dropdown,
+    trade_type_category_dropdown,
+    trade_type_dropdown,
+    values,
+    setFieldValue,
+    onHideDropdownList,
+    onChangeDropdownItem,
+    onScrollStopDropdownList,
+}: any) => {
+    
+return(<>
+    <div className='bot-builder__item'>
+        <Text>{localize('Market:')}</Text>
+        <SelectBlockField
```

Comment:
can we map this using array ? @maryia-matskevich-deriv

Reply:
@rupato-deriv yes, sure, in the next PRs, when we implement more fields and have a better idea of how to organize all of them